pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -75,6 +76,10 @@ func (b *ConsoleServerCLIConfigBuilder) CustomDeveloperCatalog(devCatalogCustomi
 	b.devCatalogCustomization = devCatalogCustomization
 	return b
 }
+func (b *ConsoleServerCLIConfigBuilder) ProjectAccessOptions(projectAccess operatorv1.ProjectAccess) *ConsoleServerCLIConfigBuilder {
```

Comment:
lets drop the `Options`
```suggestion
func (b *ConsoleServerCLIConfigBuilder) ProjectAccess(projectAccess operatorv1.ProjectAccess) *ConsoleServerCLIConfigBuilder {
```

Reply:
Updated.